src/main/kotlin/org/danilopianini/gradle/mavencentral/PublishOnCentralExtension.kt

```diff
@@ -87,24 +88,32 @@ open class PublishOnCentralExtension(val project: Project) {
         repoOwner.map { "https://github.com/$it/${project.name}" }
     )
 
+    /**
+     * The style of `javadoc` artifacts being published on Maven repositories.
+     */
+    val docStyle: Property<DocStyle> = project.objects.property<DocStyle>().convention(
+        DocStyle.HTML
+    )
+
+    /**
+     * Selects all Dokka tasks aimed at generating documentation compliant with the style declared by [docStyle].
+     */
+    internal val dokkaTasks: TaskCollection<Task>
+        get() = project.tasks.matching { task ->
+            task.name.let { it.startsWith("dokka") && it.endsWith(docStyle.get().name, ignoreCase = true) }
+        }
```

Comment:
Fetching tasks by name is very fragile. If someone defines a random task called "dokkaFoo" it would be returned -- I think it might be better to use the task type

Reply:
Fixed in b8bf993